assets/js/utils/store.ts

```diff
@@ -16,17 +16,18 @@ export default {
     }
   },
 
-  get(key) {
+  get<Value = unknown>(key: string): Value | null {
```

Comment:
Is it necessary for the `| null` to exist, since IIRC by the rules Value should already be nullable?

Reply:
I'm fairly certain `Value` in this context is not nullable by default unless we specify it here